packages/credential-provider-node/src/credential-provider-node.integ.spec.ts

```diff
@@ -474,18 +430,13 @@ describe("credential-provider-node integration test", () => {
         secretAccessKey: "STS_AR_SECRET_ACCESS_KEY",
         sessionToken: "STS_AR_SESSION_TOKEN",
         expiration: new Date("3000-01-01T00:00:00.000Z"),
-        credentialScope: "us-stsar-1__us-gov-stsar-1",
         $source: {
           CREDENTIALS_PROFILE_SOURCE_PROFILE: "o",
           CREDENTIALS_STS_ASSUME_ROLE: "i",
         },
       });
     });
 
-    xit("should prefer static credentials over role assumption metadata only if the profile is not the first one visited", async () => {
-      // TODO
-    });
```

Comment:
This test case isn't necessary, because other tests cover it.

I wrote this initially when I was going over the default credential provider chain and it looked like a distinct use-case, but isn't.